engines/bastion_katello/app/assets/javascripts/bastion_katello/capsule-content/capsule-content.routes.js

```diff
@@ -26,4 +26,12 @@ angular.module('Bastion.capsule-content').config(['$stateProvider', '$urlRouterP
         }
     });
 
+    $urlRouterProvider.otherwise(function ($injector) {
```

Comment:
That's a really good point actually.  Let me test this out and see what affects it has (if any) and rework as appropriate.